ctapipe/io/tests/test_simteleventsource.py

```diff
@@ -1,3 +1,5 @@
+""" tests of SimTelEventSource """
+# pylint: disable=import-outside-toplevel
```

Comment:
For another PR: can we disable import-outside-toplevel for all test files?

Reply:
I haven't found a way to filter by filename, but it would be nice!  We could just disable it globally since it's used sparingly and correctly for the most part.